code/controllers/subsystem/weather.dm

```diff
@@ -30,7 +30,7 @@ var/datum/subsystem/weather/SSweather
 		var/datum/weather/W = pickweight(possible_weather_for_this_z)
 		run_weather(W.name)
 		eligible_zlevels -= Z
-		addtimer(src, "make_z_eligible", rand(3000, 6000) + W.weather_duration_upper, Z) //Around 5-10 minutes between weathers
+		addtimer(src, "make_z_eligible", rand(3000, 6000) + W.weather_duration_upper, TRUE, Z) //Around 5-10 minutes between weathers
```

Comment:
I intended it to be unique. If I read the code correctly, not having it be unique could result in several ash storms at once and, while that'd be hilarious, it probably isn't good.